### PR TITLE
note: only `bundle exec ...` uses local fastlane

### PR DESCRIPTION
**Pull request by janpio**
Highlight that you have to use `bundle exec ...` for your local fastlane code checkout to be used.

### CODE REVIEW

**Review by milch — DISMISSED** [review]
™️

File: YourFirstPR.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ bundle show fastlane
 
 which should print out the path to your local development environment.
 
-From now on, every time you introduce a change to your local _fastlane_ code base, you can immediately test it by running `bundle exec fastlane …`.
+From now on, every time you introduce a change to your local _fastlane_ code base, you can immediately test it by running  (`bundle exec fastlane …`. (Note that just using `fastlane …` without `bundle exec` will **not** use your local _fastlane_ code base!)
```
Comment (milch) [inline YourFirstPR.md]: I think you snuck an extra `( ` in there at the beginning 😅

**Review by milch — CHANGES_REQUESTED** [review]

File: YourFirstPR.md (hunk 1 above)
Comment (janpio) [reply YourFirstPR.md]: Good catch 🎣 Thanks.

**Comment by janpio** [comment]
Great to see that I am not the only one confused by the review interface. I always open reviews without wanting it and submit my comments to early. Strange.

**Comment by milch** [comment]
Yeah I was just confused because I typed in the comment above, and then wanted to submit it right away with the `Request Changes` button. Turns out it doesn't work that way 😅

**Review by milch — APPROVED** [review]

**Pull request merged**